frontend/main.go

```diff
@@ -19,34 +19,30 @@ package main
 import (
 	"flag"
 	"fmt"
-	"github.com/apache/incubator-servicecomb-service-center/frontend/schema"
+
 	"github.com/astaxie/beego"
-	"log"
-	"net/http"
-	//"strconv"
 )
 
+type Config struct {
+	frontendAddr string
+	scAddr       string
+}
+
 func main() {
 	frontendIp := beego.AppConfig.String("frontend_host_ip")
 	frontendPort := beego.AppConfig.DefaultInt("frontend_host_port", 30103)
 
+	scIp := beego.AppConfig.DefaultString("httpadr", "127.0.0.1")
```

Comment:
I don't think "httpadr" and "httpport" are good names here.

Reply:
This is already used by Service Center and in my opinion we should not change the configuration name because it breaks compatiblity.

Reply:
Fixed